src/lib/middleware/instrument.ts

```diff
@@ -14,15 +14,17 @@ export default function instrument(context: Context): RequestHandler {
 
 	return (request, response, next) => {
 		const { basePath, executor } = context;
-		const wholePath = normalizePath(resolve(join(basePath, request.url)));
+		const filePath = resolve(join(basePath, request.url));
```

Comment:
Might it be better to have `shouldInstrumentFile` work with normalized paths rather than feeding it non-normalized paths?

Reply:
I started down that path, but that would mean normalizing all of the paths that come in for the instrumentation hooks, denormalizing paths for source maps, etc. and it got messy quick

Reply:
> denormalizing paths for source maps

Source map paths aren't normalized?